Taimur_COVID19_Week1

taimurkashif opened the pull request:
#### Issues Closed 
Closes #2 

Closes #3  
...

Changes proposed in this pull request:
* Presentation was completed

@kavuong

taimurkashif committed:
commiting many files

taimurkashif committed:
Added COVID19.mdx

kavuong reviewed (changes requested):
@taimurkashif can you remove all of the cache and installation files?

taimurkashif committed:
Revert "commiting many files"

This reverts commit 73b610cc73d397742799314b3b5369517892efd7.

taimurkashif committed:
Updated COVID19.mdx

Updated COVID19.mdx to the file which Joey and Jeffrey updated. It now loads in browser, with speaker notes

taimurkashif commented:
@kavuong extra files removed! Also note that the new presentation is one that @joeyhuaa and @JeffTheAggie fixed and it is now loading in browser.

kavuong reviewed (changes requested):
Please make sure your .mdx file is in the proper place and that all of the newly created assets by @joeyhuaa and @JeffTheAggie are in the right places.

Also please set up a Code Sandbox link for your branch per the guide here: https://about.bitproject.org/teams/developer-relations/setting-up-code-sandbox#for-managers-incoming-managers
